app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb

```diff
@@ -21,7 +21,7 @@
     link_title: "Learn more about fees and financial support for non-UK trainee teachers.",
     link_url: "/non-uk-teachers/fees-and-funding-for-non-uk-trainees",
   ) %>
-    <p>The figures on this page apply if you're starting your training between September 2024 and July 2025.</p>
+    <p>The figures on this page apply if you're starting your training <%= value :bursaries_fundingdates %>.
```

Comment:
Just noticed that you've deleted the closing p tag - which doesn't seem right @emmaachesongray ?

When done, do you want to get Martyn to give a 2i and then we can merge it?

Reply:
fixed that one now and will message Martyn :)